src/spyglass/utils/dj_merge_tables.py

```diff
@@ -533,13 +537,21 @@ def fetch_nwb(
             raise ValueError("Try replacing Merge.method with Merge().method")
         restriction = restriction or self.restriction or True
         merge_restriction = self.extract_merge_id(restriction)
-        sources = set((self & merge_restriction).fetch(self._reserved_sk))
+
+        if log_export and self.export_id:
+            self._log_fetch(restriction=merge_restriction)
```

Comment:
I don't think this is necessary here now.  The Merge table is logged for export by the restriction call ` self & {self._reserved_sk: source} & merge_restriction` below.

This also logs extra entries when `fetch_nwb` is called by passing a restriction for the parent source table.  In that case, `merge_restriction = True` and the entire merge table gets logged.